tox.ini

```diff
@@ -81,6 +81,7 @@ commands_pre =
   jaeger: pip install {toxinidir}/opentelemetry-sdk
   jaeger: pip install {toxinidir}/ext/opentelemetry-ext-jaeger
   opentracing-shim: pip install {toxinidir}/opentelemetry-sdk {toxinidir}/ext/opentelemetry-ext-opentracing-shim
+  zipkin: pip install {toxinidir}/opentelemetry-sdk
```

Comment:
This could be moved into a separate PR, but the zipkin exporter needs the updated SDK